engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/simple/SimpleProcessStateDefinitionManager.scala

```diff
@@ -79,6 +77,24 @@ object SimpleProcessStateDefinitionManager extends ProcessStateDefinitionManager
       .map(sa => actionStatusMap.getOrElse(sa, SimpleStateStatus.Unknown))
       .getOrElse(SimpleStateStatus.NotDeployed)
 
+  override def statusTooltip(stateStatus: StateStatus): Option[String] =
```

Comment:
I wonder is it the best place for this? Maybe in ProcessStatus? Where we create state objects?

Reply:
I will leave it now.. We'll clean it up in the future.